.travis.yml

```diff
@@ -1,4 +1,7 @@
 language: ruby
 rvm:
+  - 1.9.3
   - 2.1.0
-
+gemfiles:
```

Comment:
Ah nice.  The conventional directory is `gemfiles` and the config is `gemfile`.  Got it.